datahub-web-react/src/app/entity/group/GroupMembers.tsx

```diff
@@ -0,0 +1,67 @@
+import { Divider, List, Pagination, Row, Space, Typography } from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import { CorpUser, EntityType } from '../../../types.generated';
+import { useEntityRegistry } from '../../useEntityRegistry';
+import { PreviewType } from '../Entity';
+
+type Props = {
+    members?: Array<CorpUser> | null;
+    page: number;
+    pageSize: number;
+    totalResults: number;
+    onChangePage: (page: number) => void;
+};
+
+const MemberList = styled(List)`
+    &&& {
+        width: 100%;
+        border-color: ${(props) => props.theme.styles['border-color-base']};
+        margin-top: 12px;
+        margin-bottom: 28px;
+        padding: 24px 32px;
+        box-shadow: ${(props) => props.theme.styles['box-shadow']};
+    }
+`;
+
+const MembersView = styled(Space)`
+    width: 100%;
+    margin-bottom: 32px;
+    padding-top: 28px;
+`;
+
+export default function GroupMembers({ members, page, onChangePage, pageSize, totalResults }: Props) {
+    const entityRegistry = useEntityRegistry();
+    const list = members || [];
+
+    // todo: group membership should be paginated or limited in some way. currently we are fetching all users.
```

Comment:
I think we should be able to remove this todo right?

Reply:
yes!